test/models/questionnaire_test.rb

```diff
@@ -204,6 +204,13 @@ class QuestionnaireTest < ActiveSupport::TestCase
       assert_nil questionnaire.acc_status_author
     end
 
+    should "return nil if author deleted" do
+      user = create(:user, email: "admin@example.com")
```

Comment:
Get ready to refactor this after #241 :p
Or crap. That's gonna be my problem now...